enterprise/users/build.gradle

```diff
@@ -17,3 +17,18 @@ dependencies {
         exclude group: 'org.slf4j', module: 'jcl-over-slf4j'
     }
 }
+
+task writePropertiesFile {
+    doLast {
+        copy {
+            from "src/main/resources/"
+            into "${buildDir}/tmp"
```

Comment:
hm, is this a common pattern that we call this directory tmp? something like resources, expanded or built_templates would make it more informative